src/pattern-library/components/patterns/samples.tsx

```diff
@@ -88,10 +88,23 @@ export function SampleTableBody() {
 
 /**
  * @param {object} props
- *   @param {'sm'|'md'|'lg'} [props.size]
+ *   @param {'xs'|'sm'|'md'|'lg'} [props.size]
```

Comment:
Not required for this PR, but considering this file is already TS, perhaps we could define a proper type for the props.

Reply:
Oops. No, I should fix that. Given that I've already merged, I'll just do a quick follow-up. Thanks for spotting that!